### PR TITLE
Add a dynamic indexing test for indexing a uniform vector

### PR DESCRIPTION
**Pull request by Oletus**

### CODE REVIEW

File: sdk/tests/conformance2/glsl3/vector-dynamic-indexing.html [hunk 1]
```diff
@@ -275,6 +275,21 @@
     my_FragColor = vec4(1.0 - f, f, 0.0, 1.0);
 }
 </script>
+<script id="fshaderIndexUniform" type="x-shader/x-fragment">#version 300 es
```
Comment (kenrussell) [inline sdk/tests/conformance2/glsl3/vector-dynamic-indexing.html]: Would it be worth calling this test fShaderIndexUniformWithUniform and adding another one which indexes a global variable with a uniform? The test above indexes a local variable with a uniform, which I could imagine being handled differently.

Comment (Oletus) [inline sdk/tests/conformance2/glsl3/vector-dynamic-indexing.html]: I don't think it's worth it to exhaustively test all possible variable types, I'd rather just have this targeted regression test.

Comment (kenrussell) [inline sdk/tests/conformance2/glsl3/vector-dynamic-indexing.html]: OK.

**Comment by kenrussell** [comment]
LGTM

**Pull request merged**